src/config.rs

```diff
@@ -49,13 +52,20 @@ fn default_port() -> u16 {
     }
 }
 
-pub fn get_config() -> AWConfig {
-    let mut config_path = dirs::get_config_dir().unwrap();
+fn is_testing() -> bool {
     match Environment::active().expect("Failed to get current environment") {
-        Environment::Production => config_path.push("config.toml"),
-        Environment::Development => config_path.push("config-testing.toml"),
+        Environment::Production => false,
+        Environment::Development => true,
         Environment::Staging => panic!("Staging environment not supported"),
-    };
+    }
+}
+
+pub fn get_config() -> AWConfig {
+    let mut config_path = dirs::get_config_dir().unwrap();
+    match is_testing() {
+        false => config_path.push("config.toml"),
+        true => config_path.push("config-testing.toml")
```

Comment:
Nice with separate configs for testing and production.

Reply:
This waas the same before as well, just started using the is_testing function now instead.